modules/twinkleprotect.js

```diff
@@ -181,6 +181,9 @@ Twinkle.protect.fetchProtectionLevel = function twinkleprotectFetchProtectionLev
 		var page = protectData[0].query.pages[pageid];
 		var current = {}, adminEditDeferred;
 
+		// Save requested page's watched status for later in case needed when filing request
+		Twinkle.protect.watched = page.watchlistexpiry || page.watched === '';
```

Comment:
`page.watched` will be true or false when API formatversion=2.
```suggestion
		Twinkle.protect.watched = page.watchlistexpiry || page.watched;
```

Reply:
@Xi-Plus We're not actually using `formatversion=2` here; `Twinkle.protect.fetchProtectionLevel` uses `mw.Api`, not `Morebits.wiki.api`.  I'd planned on converting it to `formatversion=2` but didn't want to do it here when this should be a quick bugfix (I don't think there's much work needed, though).

Reply:
Oh!